internal/dao/stubs/account_dao.go

```diff
@@ -0,0 +1,79 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/pointer"
+)
+
+var notInStoreError = errors.New("the Account you've been looking for does not exist")
```

Comment:
Errors must be exported otherwise they are pretty useless. I suggest you move this to `errors.go` or elsewhere, this one can be reused by many DAO stubs.

```suggestion
var RecordNotFoundError = errors.New("DAO record does not exist")
```

Reply:
I've gone ahead and reused the `NoRowsError` to create a feel of the DAO interface... I guess we are improving :stuck_out_tongue_closed_eyes: